xknx/management/management.py

```diff
@@ -116,6 +128,44 @@ async def connection(
         finally:
             await self.disconnect(address)
 
+    async def send_broadcast(self, telegram: Telegram) -> None:
+        """Send a broadcast message."""
+
+        if isinstance(telegram.tpci, TDataBroadcast):
+            await self.xknx.cemi_handler.send_telegram(telegram)
```

Comment:
Maybe raise otherwise? 
Or is it usable / possible to only pass the payload and build the TDataBroadcast telegram here?

Reply:
Now and error is raised. I think it's cleaner to expect a Telegram here, since that also matches most of the other send/receive functions that also operate with Telegram objects.

Reply:
The P2P-class also operates with payloads. See https://github.com/XKNX/xknx/blob/88fe96ba54f5d9da3fb1dab134d74b0fb0538dff/xknx/management/management.py#L300

But this could maybe also be part of BroadcastContext while this method could be more generic to also accept other connectionless management telegrams that are not broadcasts. 

However it fits the current needs... can always be changed / extended at a later point.